cocotb/handle.py

```diff
@@ -203,7 +203,10 @@ def __iter__(self):
                     self._log.debug("Yielding index %d from %s (%s)", subindex, name, type(subhdl))
                     yield subhdl
             else:
-                self._log.debug("Yielding %s (%s)", name, handle)
+                try:
+                    self._log.debug("Yielding %s of type %s (value %s)", name, type(handle), handle.value)
```

Comment:
I think what we want to see here is the path, and we don't necessarily care about the value of the sub handles.

Reply:
This would be a change of behavior - no problem for debug output though.

Reply:
Didn't `str(handle)` print the path at one point /  for certain types of handles?

Reply:
Could be. Let's just take `_path`  now.